config/jobs/kubernetes-sigs/kwok/kwok-presubmits-main.yaml

```diff
@@ -76,8 +83,12 @@ presubmits:
     always_run: true
     spec:
       containers:
-      - image: golang:1.18
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220804-4fa19ea91a-master
```

Comment:
this looks good to me, but guess using kubekins-e2e for other jobs is an overkill?